src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx

```diff
@@ -0,0 +1,43 @@
+---
+subject: Docs
+releaseDate: '2024-04-05'
+version: '4-5-2024'
+---
+
+<Callout variant="important">
+We're trying out release notes to let you know what's changing in our docs each week. This is an experiment, so [let us know via GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose) if you find them helpful!
+</Callout>
+
+### New docs
+
+* How to guide on using the [OpenTelemetry collector to monitor SingleStore metrics](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-singlestore).
+
+### Major changes
+
+* Synthetic monitoring job manager docs now describe how to configure [user-defined variables for scripted monitors](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration) and what [job manager version](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide) supports doing so.
+* Continued rewrite of on-host integration docs to make it easier to install and ensure the docs are accurate and up to date. Up this week: [Sonarqube](/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration) and [Windows services](/docs/infrastructure/host-integrations/host-integrations-list/windows-services-integration).
+* Cross application tracing is deprecated and won't be supported in future APM agent versions. We updated the docs to reflect this.
+* Removed docs related to OpenTelemetry cumulative metrics, which was deprecated in 2023.
+
+### Minor changes
+
+* Updated API docs links related to Node.js [AI monitoring customization](/docs/ai-monitoring/customize-agent-ai-monitoring).
+* Clarified which [Data Plus features](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-prices) apply to Standard, Pro, and Enterprise accounts.
+* Clarified that when you [update the Java APM agent](/docs/apm/agents/java-agent/installation/update-java-agent) and are using one of our [Java agent APIs](/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api), you need to update the API dependencies as well.
+* Updated Python APM agent version support to 9.8.0 (from 9.9.0) to monitor [AI libraries and frameworks](/docs/apm/agents/python-agent/getting-started/compatibility-requirements-python-agent/#digital-intelligence-platform).
+* Deleted errors inbox legacy EOL announcements that are no longer relevant.
+* Updated link to ServiceNow in [destination notification](/docs/alerts-applied-intelligence/notifications/notification-integrations) instructions.
+* Our infrastructure agent supports Elastic container service collects all [EC2 node container metrics from the Docker API](/docs/infrastructure/install-infrastructure-agent/linux-installation/container-instrumentation-infrastructure-monitoring) with Cgroup version V2.
```

Comment:
Some kind of grammatical issue here I think

Reply:
Good catch. A tricky technology-based grammatical tangle :)